scss/base/4-elements/_links.scss

```diff
@@ -1,17 +1,21 @@
 a,
 .hyperlink {
-    color: $cads-palette__black;
+    color: $cads-palette__heritage-blue;
     line-height: 1.25em;
 
     &:visited {
-        color: green;
+        color: #53388D;
     }
 
     &:hover {
-        color: blue;
+        color: #002F55;
+        background-color: $cads-palette__blue-light;
     }
 
     &:active {
-        color: white;
+        color: $cads-palette__black;
+        background-color: $cads-palette__heritage-yellow;
+        text-decoration: none;
+        border-bottom: 4px solid $cads-palette__black;
```

Comment:
Maybe we should add something border width so we don't use px values?

Reply:
We will leave this for now and see if we want to have it as a variable once we have designed other interactive components.